### PR TITLE
PHC-463 TextField label

### PR DESCRIPTION
**Pull request by HaleyWardo**
**Changes**
- Conditionally rendered label and added `aria-label` to input field

Screen Shot 2021-01-28 at 2 47 58 PM

### CODE REVIEW

**Review by dexterca — APPROVED** [review]

File: src/components/TextField/TextField.test.tsx [hunk 1]
```diff
@@ -227,6 +227,24 @@ test('it renders an inverse color icon when icon and tooltipMessage are provided
...
+  expect(label).toBeInTheDocument();
+});
+
+test('it does not render label when not provided', async () => {
+  const { queryByLabelText } = renderWithTheme(
+    <TextField label="" data-testid={testId} />
+  );
+
+  const label = await queryByLabelText(/label-text/);
+  expect(label).not.toBeInTheDocument();
```
Comment (ynotdraw) [inline src/components/TextField/TextField.test.tsx:245]: Something else that may work, if you want to ensure no `label` elements at all is:

```suggestion
renderWithTheme(
  <TextField label="" data-testid={testId} />
);

expect(document.querySelectorAll('label').length).toEqual(0);
```

File: src/components/TextField/TextField.tsx [hunk 2]
```diff
@@ -200,42 +196,44 @@ export const TextField = React.forwardRef<HTMLInputElement, TextFieldProps>(
...
-              </span>
-            </Tooltip>
-          )}
-          {secondaryLabel && (
-            <span className={classes.labelSecondary}>{secondaryLabel}</span>
-          )}
-        </label>
+        {label && (
+          <label
+            aria-hidden="true"
```
Comment (ynotdraw) [inline src/components/TextField/TextField.tsx:201]: I'm curious if we can/should delete this as well?

**Review by simons5593 — APPROVED** [review]

File: src/components/TextField/TextField.test.tsx (hunk 1 above)
Comment (HaleyWardo) [reply src/components/TextField/TextField.test.tsx]: Oh sweet! I will update this.

File: src/components/TextField/TextField.tsx (hunk 2 above)
Comment (HaleyWardo) [reply src/components/TextField/TextField.tsx]: I don't think we need it anymore.

**Review by ynotdraw — APPROVED** [review]

**Pull request merged**